playbooks/roce/roce.yaml

```diff
@@ -0,0 +1,143 @@
+---
+# Install Mellanox OFED
+- import_playbook: mofed-install.yaml
+  run_once: true
+  vars:

```

Comment:
These vars should go in the default config

Reply:
Not all nodes will be with Mellanox network cards.